src/library/scala/Array.scala

```diff
@@ -499,6 +499,22 @@ object Array {
     b.result()
   }
 
+  def equals(xs: Array[AnyRef], ys: Array[AnyRef]): Boolean = {
```

Comment:
I would suggest `private[scala]`, but no strong feeling about this, so I'll yield to whatever someone else thinks.

Reply:
idk. conceivably, someone might need this in their own library.